src/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './card.css';
+
+function Card (props){ 
+    const [ email, setEmail ] = React.useState('Loading...');
+    const [ firstName, setFirstName ] = React.useState('Loading...');
+    const [ lastName, setLastName ] = React.useState('Loading...');
+    const [ avatar, setAvatar ] = React.useState('Loading...');
+    
+    React.useEffect(() => {
+        async function getData() {
```

Comment:
This function is well put together and easy to read. I'd probably extract it to a separate file/module, and then import it here. The only change you'd need to make is to have it take in the `userId` as an argument. The function could return the `obj` from the last line. Then here, in your effect, you can attach a `then` callback to update the state.